checkpoint/cp-resource-management/templates/deployment.yaml

```diff
@@ -61,7 +61,7 @@ spec:
             allowPrivilegeEscalation: false
           env:
             - name: DOME9_URL
-              value: "https://api-cpx.dome9.com"
+              value: {{ include "dome9.url" . | quote }}
```

Comment:
The function already has quotes - do we need to quote here as well?
Does it have an impact?

Reply:
BTW, we don't need quotes for URL at all I think. I don't remember who pointed out for me on this - but I removed it in ea branch

Reply:
no impact. I think we might have unneeded quotes in several places, actually.